src/Polysemy.hs

```diff
@@ -41,7 +41,7 @@ module Polysemy
     --
     -- @
     -- writeLine :: 'Member' Console r => String -> 'Sem' r ()
-    -- readLine  :: 'Member' Console r => 'Sem' r String
+    -- readLine  :: 'Member' Console r => 'Sem' r 'String'
```

Comment:
This one is intentionally omitted; i'd rather just keep the links here for polysemy concepts.